sycl/source/detail/stream_impl.cpp

```diff
@@ -61,6 +61,29 @@ size_t stream_impl::get_size() const { return BufferSize_; }
 
 size_t stream_impl::get_max_statement_size() const { return MaxStatementSize_; }
 
+void stream_impl::fill(QueueImplPtr Queue) {
+  auto Q = detail::createSyclObjFromImpl<queue>(Queue);
+  Q.submit([&](handler &cgh) {
+    auto StreamBuf =
+        detail::Scheduler::getInstance().StreamBuffersPool.find(this);
+    assert((StreamBuf !=
+            detail::Scheduler::getInstance().StreamBuffersPool.end()) &&
+           "Stream is unexpectedly not found in pool");
+    auto &FlushBuf = StreamBuf->second->FlushBuf;
+    // Only size of buffer_impl object has been resized.
+    // Value of Range field of FlushBuf instance is still equal to
+    // MaxStatementSize only.
+    size_t FlushBufSize = detail::getSyclObjImpl(FlushBuf)->get_count();
+    auto FlushBufAcc = FlushBuf.get_access<access::mode::read_write,
```

Comment:
Please, use `host` target for accessor for non-interop host-task.
Despite of permitted device-like accessor for non-interop host-task it's still prohibited by specification.

This is a known issue.

Reply:
Done